src/extra/AutoUIApp/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Navbar } from './Navbar';
+import { Sidebar } from './Sidebar';
+import { Router, Switch, Route } from 'react-router-dom';
+import { Content } from './Content';
+import { createGlobalStyle } from 'styled-components';
+import { Provider } from '../..';
+import { history } from '../AutoUI/utils';
+import { Flex } from '../../components/Flex';
+import { Box } from '../../components/Box';
+import { OpenApiJson } from './openApiJson';
+
+const GlobalStyle = createGlobalStyle`
+	html,
+	body {
+		height: 100%;
+		margin: 0;
+		padding: 0;
+	}
+`;
+
+// tslint:disable-next-line no-namespace
+declare global {
+	interface Window {
+		REACT_APP_API_HOST: string;
+		REACT_APP_TITLE: string;
+	}
+}
+
+export interface AutoUIAppProps {
+	openApiJson: OpenApiJson;
+	title: string;
+	logo?: string;
+}
+
+export const AutoUIApp = ({ openApiJson, title, logo }: AutoUIAppProps) => {
+	return (
+		<Provider>
+			<Router history={history}>
+				<GlobalStyle />
+				<Navbar title={title} logo={logo} />
+				<Flex>
+					<Sidebar
+						width="166px"
+						openApiJson={openApiJson}
+						isCollapsed={false}
+					/>
+					<Box
+						width="calc(100% - 166px)"
+						height="calc(100vh - 60px)"
```

Comment:
A comment here about the 60px being subtracting being the header height might be nice :)